### DISCUSSION TITLE
Bootstrap Controls For Tailwind

### DISCUSSION THREAD

**Discussion by ImAbhishekTomar** [tailwindlabs/tailwindcss#6307] [Ideas]
👋,

 I am planning to recreate all bootstrap controls for tailwind. 
Please let me know if something is already there or the tailwind team is already working on something like this.

- Thanks

**Comment by billriess** [comment]
There is https://tailwindui.com and https://headlessui.dev if that's what you mean.

**Comment by anburocky3** [comment]
As said by @billriess `Tailwind UI` component has these, but only a few are free. Most of them are purchasables. You can start creating it and give it a try. If it is modern-looking, I will be using yours.

**Comment by Womack52** [comment]
Satisfied to consider your to be as I would assess I have an indistinguishable issue, I am also puzzled and requiring light on this ill defined issue. Need assistance.
[Skylight Paycard](https://www.skylightpaycard.me/)
